apps/src/templates/tables/QuickActionsCell.story.jsx

```diff
@@ -25,5 +25,26 @@ export default storybook => {
             </QuickActionsCell>
           )
         },
+        {
```

Comment:
- [ ] name these more clearly so that they better reflect the difference between a header cell and a body cell